DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UISegmentRecorder.swift

```diff
@@ -34,7 +34,8 @@ internal struct UISegmentRecorder: NodeRecorder {
                 }
             }()
         )
-        return SpecificElement(wireframesBuilder: builder, subtreeStrategy: .ignore)
+        let node = Node(viewAttributes: attributes, wireframesBuilder: builder)
+        return SpecificElement(subtreeStrategy: .ignore, nodes: [node])
```

Comment:
feels like we could have convenience function that would allow us to initialise the `SpecificElement` with attributes, builder and subtree strategy 🤔

Reply:
THB I don't think this is good idea. Except saving few lines of code (in entire PR) this won't achieve any architectural goal, but only make things more coupled. `SpecificElement` would have to additionally depend on `ViewAttributes` and `NodeWireframesBuilder` - even if we call it convenience function, that would count as dependency and make concept less readable: _Why passing strategy, attributes and builder to some `SpecificElements` vs strategy and nodes to others?_